spec/visual/samples/raml

```diff
@@ -0,0 +1,194 @@
+#%RAML 1.0
+---
+title: Jukebox API
+baseUri: http://jukebox.api.com
+version: v1
+
+types:
+ song: !include jukebox-include-song.schema
+ artist: !include jukebox-include-artist.schema
+ album: !include jukebox-include-album.schema
+
+
+resourceTypes:
```

Comment:
Is all of this sample specific to RAML, or is it just YAML? I'm inclined to think we don't need such a large sample given this is a very small subclass of YAML. YAML includes a huge sample. If this sample is very indicative of RAML then it's OK.

Reply:
I echo this point. The visual sample here can be very short.